doc/ratbagd.8

```diff
@@ -0,0 +1,19 @@
+.TH ratbagd 8 "Apr 13, 2016" ratbagd
+.SH NAME
+ratbagd \- system daemon to introspect and modify configurable mice
+.SH SYNOPSIS
+.B ratbagd
+.RB [ \-\-verbose ]
+.SH DESCRIPTION
+.B ratbagd
+starts the daemon. It shouldn't be invoked directly;
+.B ratbagd
+is normally started through DBus activation.
+.SH SEE ALSO
+.BR ratbagctl (1)
+.SH AUTHORS
+.B ratbagd
+was written by David Herrmann, Peter Hutterer and Benjamin Tissoires.
+.PP
+This manual page was written by Stephen Kitt <skitt@debian.org> for
+the Debian GNU/Linux system (but may be used by others).
```

Comment:
I'd drop `(but may be used by others)` since it is covered by the same license as ratbagd itself.